### DISCUSSION TITLE
Startup Prioriy

### DISCUSSION THREAD

**Discussion by kms107** [rocksdanister/lively#1191] [Q&A]
Lively normally startup very late in my system. it starts almost in the end after the other startup programs has completed. 
How can I get startup priority or sequence of lively during Windows bootup?

I have checked my registry but I cannot see key for lively. I can soo only other startup program keys for others.

**Comment by rocksdanister** [comment]
You can find the information here: https://github.com/rocksdanister/lively/issues/499
